src/main/resources/config/application.yml

```diff
@@ -97,12 +97,18 @@ spring:
       hibernate.cache.use_second_level_cache: true
       hibernate.cache.use_query_cache: false
       hibernate.generate_statistics: false
+      
+      
       # modify batch size as necessary
       hibernate.jdbc.batch_size: 25
```

Comment:
Preferable to use **_yml_** way of declaring these configurations _instead of old **.properties** style_, since root term is getting repeated(_hibernate_)